app/views/groups/index.html.erb

```diff
@@ -49,33 +49,7 @@
     <%= link_to t('upload'), 'javascript:void(0);', onclick:'modal_upload.open();' %>
 <% end %>
 
-<div id="notice">
-  <div class="success" id="add_success" style="display: none;"></div>
-  <% # This is for error display injected by JavaScript %>
-
-  <div class="error" id="errors" style="display: none; ">
-    <h3><%= t("groups.problem") %></h3>
-    <div id="errors_title"></div>
-    <div id="errors_contents"></div>
-  </div>
-
-  <div class="warning" id="warnings_group" style="display: none; ">
-    <h3><%= t("groups.warning") %></h3>
-    <div id="warnings_group_msg"></div>
-  </div>
-
-  <div class="warning" id="warnings_grace" style="display: none; ">
-    <h3><%= t("groups.warning") %></h3>
-    <div id="warnings_grace_msg"></div>
-  </div>
-
-  <div class="warning" id="global_action_warning" style="display: none; ">
-    <h3><%= t("groups.warning") %></h3>
-    <div id="global_action_warning_msg"></div>
-  </div>
-  <div id="job-messenger"></div>
-
-</div>
+<%= render 'shared/flash_message' %>
```

Comment:
This is true, this render make the page has two duplicated flash message div. I just delete this one. But the one already included in the template has layout issue which will show the flash message on the bottom of the page